server/app/routes/users.router.js

```diff
@@ -8,8 +8,17 @@ var Game = db.Game;
 var Task = db.Task;
 var Event = db.Event;
 
+
+//For use in autocomplete to invite players
+router.get('/invite', function(req,res, next){
```

Comment:
Oh wait I just read down a little further and I see that an invite is a "type" of user.